From 7dac4e242934831fbe7772c61b1af46c2a5cac4c Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Tue, 20 Jun 2017 17:39:22 +0800
Subject: [PATCH 3031/5242] MLK-15110-11 arm64: dtsi: fsl-imx8qxp: Add DPR0/1
 irq resources for DPU

commit  3884b7378982ea7baa0349169c36f306c4a8c059 from
https://source.codeaurora.org/external/imx/linux-imx.git

The Display Prefetch Resolve(DPR) engine is the prefetch engine of DPU.
This patch adds the DPR0/1's irq resources for DPU.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index f98e035..1f7a6cc 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -967,7 +967,9 @@
 			     <GIC_SPI 44 IRQ_TYPE_LEVEL_HIGH>,
 			     <GIC_SPI 45 IRQ_TYPE_LEVEL_HIGH>,
 			     <GIC_SPI 46 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 47 IRQ_TYPE_LEVEL_HIGH>;
+			     <GIC_SPI 47 IRQ_TYPE_LEVEL_HIGH>,
+			     <GIC_SPI 50 IRQ_TYPE_LEVEL_HIGH>,
+			     <GIC_SPI 51 IRQ_TYPE_LEVEL_HIGH>;
 		interrupt-names = "irq_common",
 				  "irq_stream0a",
 				  "irq_stream0b",	/* to M4? */
@@ -975,7 +977,9 @@
 				  "irq_stream1b",	/* to M4? */
 				  "irq_reserved0",
 				  "irq_reserved1",
-				  "irq_blit";
+				  "irq_blit",
+				  "irq_dpr0",
+				  "irq_dpr1";
 		clocks = <&clk IMX8QXP_DC0_PLL0_CLK>,
 			 <&clk IMX8QXP_DC0_PLL1_CLK>,
 			 <&clk IMX8QXP_DC0_DISP0_CLK>,
-- 
1.7.9.5

